SEM_2_PRACTICALS/prac_08/taxi_test.py

```diff
@@ -0,0 +1,8 @@
+from SEM_2_PRACTICALS.prac_08.taxi import Taxi
+
+Prius = Taxi(100, "Prius 1")
```

Comment:
This name is misleading. Since Prius starts with a Capital, we think it's a class, but it's an object.